src/pages/Main/Components/Promotion.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const Promotion = ({ id }) => {
+  return (
+    <Wrapper>
+      <img alt="promotion" src={Promotion_Arr[id]} />
+    </Wrapper>
+  );
+};
+
+export default Promotion;
+
+const Wrapper = styled.div`
+  width: 890px;
+  img {
+    height: 370px;
+    width: 890px;
+    border-top-right-radius: 5px;
+    border-bottom-right-radius: 5px;
+  }
+`;
+
+const Promotion_Arr = [
```

Comment:
```suggestion
const PROMOTIONS = [
```
- 상수데이터는 모두 UPPER_CASE + snake_case로 작성해주세요!
- Arr등의 데이터타입이 변수명에 붙어있는 건 좋지않습니다! 프로모션이 여러개들어있다라는 의미는 복수형 표현을 통해서 충분히 표현 가능합니다!

Reply:
네 수정했습니다!